pkg/arvo/app/publish.hoon

```diff
@@ -1376,7 +1381,10 @@
     =/  pax=path  /app/publish/notebooks/[book.act]/publish-info
     :_  state
     [(write-file pax %publish-info !>(new-info)) cards]
-  ::  %edit-note
+  ::  %edit-note:
+  ::    If poke is from us, eagerly store new note in books, and place the old
+  ::    note in limbo. If poke is to us, save file, otherwise forward the poke.
+  ::    If forwarded poke fails, old note is restored from limbo.
```

Comment:
This helps clarify what this is used for. Would've also liked to see comments about the logic flow break up the section of code that follows, but this gives good context at least.

Limbo-based revert behavior for these edits is very straightforward to understand. But then you look at the create cases:

> Failed adds remove the thing from our regular state and put it in limbo. Currently there's no way non-involved way of getting it back out of limbo, but I figured it would be better to at least store it and give savvy users that option than just swallowing it.

If that's the angle you want to take (prevent loss of changes/data), shouldn't edits, on-restore, be _swapped_ with the old version from limbo, instead of _replaced_?

Arguably, at that point, you get very close to wanting full-on revision control, which is clearly out of scope here...